src/gpg_ux_nanos.c

```diff
@@ -197,7 +197,6 @@ unsigned int ui_uifconfirm_nanos_button(unsigned int button_mask, unsigned int b
         }
       }
       FINALLY {
-        G_gpg_vstate.UIF_flags = 0;
```

Comment:
Why this remove. UIF must be confirmed for each operation.

Reply:
Sorry. 
Wrongly commited this change. I was used it while investigation. Tested it without this change. UIF is working for Decrypting, Signing. I have not tested internal authentication.